### DISCUSSION TITLE
How can i use this module?

### DISCUSSION THREAD

**Discussion by czriad001** [PlatformCommunity/flysystem-bunnycdn#10] [Q&A] [answered]
Hello,

I purchased laravel application from codecanyon, I want to use this bunnycdn module. Can you please tell me? how can I do that?

Thank you

**Comment by sifex** [comment, marked as answer]
Hey @czriad001, I've updated the README with instructions on how to install BunnyCDN Flysystem Adapter into a Laravel Application :)
